ouster_client/src/client.cpp

```diff
@@ -154,6 +154,90 @@ SOCKET udp_data_socket(int port) {
     return SOCKET_ERROR;
 }
 
+
+SOCKET mtp_data_socket(int port, const std::string& udp_dest_host = "",
```

Comment:
@Mattc11 I think you are quite familiar multicast, mind take a look at this?

Reply:
It looks good to me, only comment is the INADD_ANY.

Reply:
Done in [6a42349](https://github.com/ouster-lidar/ouster_example/pull/480/commits/6a42349849740291551c11b631c8424fee9732e3)